Makefile

```diff
@@ -81,10 +86,19 @@ localeextract:
 	python manage.py extract
 	python manage.py merge
 
+locale-populate-react:
+	@ mkdir -p locale/compendia
+	for localename in `find locale -mindepth 1 -maxdepth 1 -type d | cut -d/ -f2 | grep -v templates | grep -v compendia`; do \
+		rm -f locale/compendia/$$localename.compendium; \
+		msgcat --use-first -o locale/compendia/$$localename.compendium locale/$$localename/LC_MESSAGES/django.po locale/$$localename/LC_MESSAGES/javascript.po; \
+		rm -f locale/$$localename/LC_MESSAGES/react.po; \
```

Comment:
If a React component had used a new string, is it possible that this react.po file being deleted here could contain a translation of that new string that does not appear in the compendium? If so, then this deletion might be a problem...  Would it work to use this react.po (or a copy of it) instead of /dev/null in the msgmerge line below so that any React-only translations are not lost?